specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-09-01-preview/examples/alertRuleTemplates/GetAlertRuleTemplates.json

```diff
@@ -31,6 +31,7 @@
               "lastUpdatedDateUTC": "2020-02-27T00:00:00Z",
               "createdDateUTC": "2019-02-27T00:00:00Z",
               "status": "Available",
+              "version": "1.0.1",
```

Comment:
>1.0.1 [](http://example.com/codeflow?start=26&length=5)

Shouldn't it be <1.0.1> per the description in the spec.

Reply:
No, when I write <X> I mean that the thing inside the <> is the actual content.